distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -391,6 +391,14 @@ grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldap
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}attributeMap.properties", "read";
 }
 
+grant codeBase "file:/org.apache.karaf.jaas.command" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read";
+}
```

Comment:
✏️ We could tack this module onto another similar grant block. The only additional permission it would get is read access to users.attributes, which I don't think is an issue.  @stustison what do you think?

https://github.com/blen-desta/ddf/blob/5b384c1f7e9a39f00e4b38bb7c5b543f54585e46/distribution/ddf-common/src/main/resources/security/default.policy#L229-L232